eng/packages/http-client-csharp/generator/Azure.Generator/src/NamespaceVisitor.cs

```diff
@@ -47,7 +47,7 @@ internal class NamespaceVisitor : ScmLibraryVisitor
             }
             else
             {
-                type.Type.Namespace = AzureClientPlugin.Instance.TypeFactory.RootNamespace;
+                type.Type.Namespace = AzureClientPlugin.Instance.TypeFactory.GetCleanNameSpace(AzureClientPlugin.Instance.TypeFactory.PackageName);
```

Comment:
I don't think we need to call GetCleanNameSpace here.

Reply:
This is needed to handle the namespace conflicting with models.
For instance, there is model called Type, and then the namespace should be `_Type`

Reply:
But isn't this already handled in PackageName property?

Reply:
That's true. 
But without these, the spector tests are failing, I need to take a closer look at this.

Reply:
Removed, didn't get the failure this time...